configure.ac

```diff
@@ -42,6 +42,17 @@ AC_PROG_LIBTOOL
 
 AC_SUBST(ASSEMBLER_WARN_CFLAGS)
 
+# For splash screen dummy compositor
+AC_ARG_ENABLE(dummy-compositor,
+  AS_HELP_STRING([--enable-dummy-compositor],
```

Comment:
The issue I see here is that each backend is mutually exclusive. What would happen if I passed both "--enable-vulkan" and "--enable-dummy-compositor"? I think a better approach would be to have a '--with-compositor=' option that could take one of "gl,vulkan,dummy", at least until we get runtime selection working.

Reply:
hmm good point. @csravelar Can you do that change ?

Reply:
Or other way around would be to have checks that only one backend is requested and throw out an error if thats not the case.